Lib/plistlib.py

```diff
@@ -888,6 +888,11 @@ def loads(value, *, fmt=None, dict_type=dict):
     """Read a .plist file from a bytes object.
     Return the unpacked root object (which usually is a dictionary).
     """
+    if isinstance(value, str):
+        if fmt == FMT_BINARY:
+            msg = "value must be bytes-like oebject when fmt is FMT_BINARY"
+            raise TypeError(msg)
```

Comment:
```suggestion
            raise TypeError("value must be bytes-like object when fmt is FMT_BINARY")
```

The *msg* variable is not necessary (also fixes a typo in the message itself)

Reply:
Thanks for the review, I fixed the typo, but kept the msg variable. It's here to make the line length shorter than 80.

Reply:
I still don't like introducing a variable for the message here.  You could do this instead:

```python
            raise TypeError(
                        "value must be bytes-like object when fmt is FMT_BINARY"
             )
```

If I counted correctly this just about fits into 80 characters. Alternatively split the string:

```python
            raise TypeError(
                        "value must be bytes-like object "
                         "when fmt is FMT_BINARY"
             )
```

Reply:
Got it, updated.